snap/snapcraft.yaml

```diff
@@ -18,7 +18,7 @@ title: NVIDIA DCGM
 
 apps:
   dcgm-exporter:
-    command: bin/dcgm-exporter
+    command: bin/dcgm-exporter -f $SNAP/dcgm-custom-metrics.csv
```

Comment:
This isn't a daemon; the user will directly run `dcgm-exporter`, possibly providing more cli args, so it seems strange to hardcode an argument here.  How about:

- ship our custom extended metrics list with the snap
- don't hardcode args to dcgm-exporter command
- document how to use the extended metrics

Alternatively, make dcgm-exporter a daemon also, and then we can restrict the arguments, and possibly use snap config so the user can select a metrics list (eg. default, extended, etc.).

Reply:
We plan to make it a daemon; it's already in progress in the other PR #4. Yes, we can use snap config for that

Reply:
We could keep it there so that when we decide to make the metric list configurable, the lists are there. It does not hurt to have them, what do you think?

Reply:
I also saw that `dcgm-exporter` needs them in that directory as a fallback